std_msgs/Header header
int32 num_of_lanes
geometry_msgs/Point[] lane
float32[] num_of_points
sensor_msgs/Image frame
